static/js/core.js

```diff
@@ -1,226 +1,89 @@
-/*jslint regexp: true */
-/*global YUI, core, browser, window, Modernizr, localStorage */
-
 /**
  * Ubuntu Core Front-End Framework
  *
  * Core javascript file part of Ubuntu Core Front-End Framework
  *
- * This file containes the classes required by ubuntu.com to interact.
- *
  * @project        Ubuntu Core Front-End Framework
  * @author         Web Team at Canonical Ltd
- * @copyright      2012 Canonical Ltd
+ * @copyright      2016 Canonical Ltd
```

Comment:
As previously discussed, could we remove the year here? I believe it weakens our copyright if anything.